SparkPost/matchbox
# Remove Sass

### What Changed
- Removes Sass and CSS related dependencies in the workspace and matchbox
- Removes CSS plugins from the rollup build

### How To Test or Verify
- Run Storybook, verify everything works
- Run the Matchbox build, verify the build succeeds

### PR Checklist
